modules/ROOT/pages/cloud/overview/overview.adoc

```diff
@@ -0,0 +1,94 @@
+:page-layout: api-partial
+
+The Redpanda Cloud API is a collection of REST APIs that allow you to interact with different parts of Redpanda Cloud. To understand the Cloud API basics, see the xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-overview.adoc[Redpanda Cloud API Overview]. To get started using the API, try the <<quickstart,Quickstart>>.
+
+== Control Plane API
+
+The Control Plane API enables you to programmatically manage your clusters, networks, and resource groups.
+
+The *base URL* of the Control Plane API for all organizations is:
+
+```
+https://api.redpanda.com
+```
+
+== Data Plane APIs
+
+The Data Plane APIs enable you to programmatically manage the resources within your clusters, including topics, users, access control lists (ACLs), and connectors.
+
+The *base URL* of the Data Plane APIs is unique to every cluster. You can retrieve this value by making a xref:get-/v1beta2/clusters/-id-[Get Cluster] request to your target cluster. Use the `dataplane_api.url` from the response body as the base URL when calling the Data Plane API endpoints. 
+
+== Quickstart
+
+. Authenticate to the API. See <<use-this-page-to-try-the-cloud-api>> if issuing requests on this page, or see the xref:ROOT:deploy:deployment-option/cloud/api/cloud-api-authentication.adoc[authentication guide] for details.
+. Create a resource group by making a xref:post-/v1beta2/resource-groups[`POST /v1beta2/resource-groups`] request.
+. Create a network by making a xref:post-/v1beta2/networks[`POST /v1beta2/networks`] request. When the long-running operation is complete, proceed to the next step.
+. Create a cluster by making a xref:post-/v1beta2/clusters[`POST /v1beta2/clusters`] request.
+. For BYOC, run `rpk cloud byoc`, passing the `metadata.cluster_id` from the Create Cluster response as a flag:
++
+GCP:
++
+```bash
+rpk cloud byoc gcp apply --redpanda-id=<metadata.cluster_id> --project-id=<gcp-project-id>
+```
++
+AWS:
++
+```bash
+rpk cloud byoc aws apply --redpanda-id=<metadata.cluster_id>
+```
+
+See also:
+
+- xref:ROOT:deploy:deployment-option/cloud/api/cloud-controlplane-api.adoc[Manage Clusters Using the Cloud API]
+- xref:ROOT:deploy:deployment-option/cloud/api/cloud-dataplane-api.adoc[Manage Cluster Resources Using the Cloud API]
+
+== Use this page to try the Cloud API
+
+Before you can issue requests against the API from your browser, make sure you follow these steps:
+
+. Go to *Authentication* in the sidebar.
+. Click the *Get Token* button. Do not edit the input field next to the button.
```

Comment:
```suggestion
. Click the *Get Token*n. Do not edit the input field next to the button.
```

Reply:
Check out https://github.com/redpanda-data/docs-site/blob/main/meta-docs/STYLE-GUIDE.adoc#gui-style-guidelines
The Style guideline is not to name the GUI element--these elements are named throughout this topic--need to clean those up.